### DISCUSSION TITLE
全家重訊揭曉新股東 國泰蔡家確定入主

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#941] [General]
上次因為龍邦想併購泰山，於是泰山就把他最值錢的全家股權賣了，國泰金買了很多，變成全家的第二大股東，我其實有點驚訝國泰會去購買畢竟他是金融業，感覺跟零售業沒什麼關係，可能是未來有什麼想合作的吧。
網址: https://ctee.com.tw/news/industry/773480.html

**Comment by CPCpyt** [comment]
我認為全家本身是一個獲利能力非常穩定的公司，
擁有其股份相當於擁有非常穩定的金雞母，
就連龍邦想併購泰山很大一部分原因也是為了全家股份，
國泰的入主能夠入主我認為是合理而且是非常有能力。

**Comment by CPCpyt** [comment]
另外，我想這會讓泰山的股價更直接反應其自身的獲利能力，
也可以發現近期泰山的股價波動性也是非常大，對象套利的投資人未必是壞事。

最後綜觀總經環境，當前是高利率、經濟即將步入衰退。
比起獲利，能夠穩定且持續成長是非常重要的選股指標，
我也在近期全部清倉，等待重新進場的時機。
